examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -78,8 +78,10 @@ const SUPPORTS_STANDARDS: [StandardIdentifier<'static>; 3] =
 const SUPPORTS_PERMIT_ENTRYPOINTS: [EntrypointName; 2] =
     [EntrypointName::new_unchecked("updateOperator"), EntrypointName::new_unchecked("transfer")];
 
-/// Tag for the CIS3 Registration event.
+/// Tag for the Registration event.
 pub const REGISTRATION_EVENT_TAG: u8 = 0u8;
+/// Tag for the CIS3 Nonce event.
+pub const NONCE_EVENT_TAG: u8 = u8::MAX - 5;
```

Comment:
Why did you choose this value for it? I don't think this'll work since the serialization is derived, so the tag of the "Nonce" variant below will be 1, not 250.

Reply:
Right, I fixed the `Serial`. The CIS-3 standard defines `250` as tag for this event.